src/test/test-argv-util.c

```diff
@@ -115,6 +115,21 @@ TEST(rename_process) {
         test_rename_process_one("1234567", 1); /* should always fit */
 }
 
+TEST(argv_help) {
+        char *args_a[] = {(char*)"program", NULL}; /* Test case 1: No argument */
+        assert_se(argv_looks_like_help(1, args_a) == true);
```

Comment:
You can do this as follows:

```
assert_se(argv_looks_like_help(1, STRV_MAKE("program")));
```

Ditto for all the other tests

Reply:
Wow, Thank you so much sir. I was already getting worried, my tests were failing on GitHub. Plus, I prefer the neatness of this option too. Awesome!